src/pages/program/[slug]/index.tsx

```diff
@@ -7,23 +7,18 @@ import AuthenticationModal from "../../../components/Authentication/Authenticati
 import ErrorScreen, { ErrorScreenType } from "../../../components/ErrorScreen";
 import {
   defaultContentState,
-  EditorProvider,
-  PublishButton,
   ReadOnlyTextEditor,
-  TextEditor,
-  ToolBar,
 } from "../../../components/RichTextEditing";
 import {
   PageGetProgramBySlugComp,
   ssrGetProgramBySlug,
 } from "../../../generated/page";
 import { AuthorizationLevel, useAuthorizationLevel } from "../../../hooks";
-import SignedInAsIndicator from "../../../layouts/SignedInAsIndicator";
 import ChooseTabLayout from "../../../layouts/ChooseTabLayout";
 import PageContainer from "../../../layouts/PageContainer";
+import SignedInAsIndicator from "../../../layouts/SignedInAsIndicator";
```

Comment:
Opinion: We should take this out because we have the leftside tab that shows you're logged in. Although I know Serene added it :0

Reply:
Noted, but let's stick with what design says for now.